samples/benchmark_test/benchmark_test.cc

```diff
@@ -58,12 +60,29 @@ int main(int argc, char* argv[]) {
         out_image_c = atoi(argv[kOutChannel]);
         out_image_h = atoi(argv[kOutImageH]);
         out_image_w = atoi(argv[kOutImageW]);
+        is_depthwise = atoi(argv[deptwise_option]);
+        switch(is_depthwise){
+            case 1:
+                multiplier = out_image_c/in_image_c;
```

Comment:
You already calculated the multiplier on line 48, why do it again?

Reply:
That multiplier is the result calculated by the default value, and this multiplier is the result of reading the command line and then calculating